packages/palette/src/svgs/index.tsx

```diff
@@ -93,3 +93,5 @@ export * from "./VerifiedIcon"
 export * from "./WatchingIcon"
 export * from "./WeChatIcon"
 export * from "./XCircleIcon"
+export * from "./EarnMoreIcon"
```

Comment:
Can we alphabetize this import?

Reply:
@damassi by alphabetize, do you mean put in alphabetical order??

Reply:
Yup; the rest of the file has icon names in order.